server/routes/utilMongoose.js

```diff
@@ -97,9 +97,9 @@ const createAlumni = async (_email, _name, _country, _city, _profession, _compan
             country: _country,
             city: _city,
             jobTitle: _profession,
```

Comment:
`jobTitle` and `company` are optional fields on the alumni schema and removing the struct requirement makes setting up test data a wee bit less annoying.